CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+## Contributing
+
+1. Fork it
+2. Create your feature branch (`git checkout -b my-new-feature`)
+3. Commit your changes, including tests (`git commit -am 'Added some feature'`)
+4. Push to the branch (`git push origin my-new-feature`)
+5. Create new Pull Request, and mention @wvanbergen.
```

Comment:
I am happy to let you maintain this thing. I have not worked in this space for a long long time :)